.changeset/lazy-zebras-pay.md

```diff
@@ -0,0 +1,18 @@
+---
+'@backstage/create-app': patch
+---
+
+Added some instruction comments to the generated config files, to clarify the
+usage of `backend.baseUrl` and `backend.listen.host`.
+
+Updated the builtin backend Dockerfile to honor the `app-config.production.yaml`
+file. If you want to do the same, change `packages/backend/Dockerfile` as follows:
+
+```diff
+-COPY packages/backend/dist/bundle.tar.gz app-config.yaml ./
++COPY packages/backend/dist/bundle.tar.gz app-config*.yaml ./
+ RUN tar xzf bundle.tar.gz && rm bundle.tar.gz
+
+-CMD ["node", "packages/backend", "--config", "app-config.yaml"]
++CMD ["node", "packages/backend", "--config", "app-config.yaml", "--config", "app-config.production.yaml"]
+```
```

Comment:
Could perhaps make sense to call out the `host: 0.0.0.0` addition too? and ofc the `.dockerignore` change